stitcher/test/gtest/images.cpp

```diff
@@ -0,0 +1,201 @@
+#include "gtest/gtest.h"
+#include "airmap/images.h"
+#include "airmap/logger.h"
+#include "airmap/panorama.h"
+
+using airmap::stitcher::GeoImage;
+using airmap::stitcher::Panorama;
+using airmap::stitcher::SourceImages;
+using airmap::filesystem::path;
+using airmap::logging::Logger;
+using airmap::logging::stdoe_logger;
+
+class SourceImagesTest : public ::testing::Test
+{
+protected:
+    void SetUp() override
+    {
+        path image_directory = (path("../") / path("test") / path("fixtures") /
+                                path("panorama_aus_1"));
```

Comment:
You only need to construct the path for the first string literal. Path supports `operator /` for string literals and will return another path, so path objects may be constructed like such:

```suggestion
        path image_directory = path("../") / "test" / "fixtures" / "panorama_aus_1";
```

Reply:
I've added a new operator on `path` that accepts a string literal and changed all of these.

Reply:
Ha, now I see that you were talking about `boost::filesystem::path`.  I went ahead and removed `airmap::filesystem` and replaced it with `boost::filesystem`.